docs/qemu.md

```diff
@@ -31,19 +32,36 @@ It will stop on the QEMU console, to continue, simply
 ## 3.2 ARMv8-A
 It won't stop, it will just boot up QEMU and OP-TEE.
 
-#4. Host-Guest folder sharing in QEMU/QEMUv8 configurations
+# 4. Host-Guest folder sharing
 
 To avoid changing rootfs CPIO archive each time you need to add additional files
 to it, you can also use VirtFS QEMU feature to share a folder between the guest
 and host operating systems. To use this feature enable VirtFS QEMU build in
-build/common.mk (set QEMU_VIRTFS_ENABLE ?= y), adjust QEMU_VIRTFS_HOST_DIR and
+build/common.mk (set `QEMU_VIRTFS_ENABLE ?= y`), adjust `QEMU_VIRTFS_HOST_DIR` and
 rebuild QEMU.
 
 To mount host folder in QEMU, simply run:
 
 ```bash
 $ mount_shared <mount_point>
 ```
+# 5. SLiRP user networking
+To enable SLiRP user networking just set `QEMU_VIRTFS_ENABLE ?= y` in `common.mk`.
+After booting QEMU VM, you can receive IP address via DHCP for eth0 interface by
+running:
```

Comment:
Ha! Yes! I didn't even remember ;o
My script is missing the device detection part, however. It can be safely modified to add ifconfig I think.